core/parallel_state_processor.go

```diff
@@ -304,6 +304,11 @@ func (p *ParallelStateProcessor) Process(block *types.Block, statedb *state.Stat
 		metadata = true
 	}
 
+	if !VerifyDeps(deps) {
```

Comment:
If deps is nil or empty, we can skip the check.

Reply:
[Done](https://github.com/maticnetwork/bor/pull/841/commits/8fe290164e80634a5c9776464f0e99aa2e86ba5e), thanks.